certified-connectors/JIRA/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2183 @@
+                    {
+                        "description": "Returned if the task is not found.",
+                        "schema":
+                        {
+                            "type": "array",
+                            "items":
+                            {
+                                "type": "string"
+                            }
+                        },
+                        "headers": {}
+                    }
+                }
+            }
+        },
+        "/3/user":
+        {
+            "get":
+            {
+                "description": "Returns a user. Permissions required: Browse users and groups.",
+                "summary": "Get User",
+                "tags":
+                [
+                    "Users"
+                ],
+                "operationId": "getUser",
+                "deprecated": false,
+                "produces":
+                [
+                    "application/json"
+                ],
+                "parameters":
+                [
+                    {
+                        "name": "accountId",
+                        "in": "query",
+                        "required": true,
+                        "type": "string",
+                        "maxLength": 128,
+                        "description": "The account ID of the user, which uniquely identifies the user across all Atlassian products. For example, *5b10ac8d82e05b22cc7d4ef5*. Required.",
+                        "x-ms-summary": "Account ID"
+                    },
+                    {
+                        "name": "expand",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "description": "Use expand to include additional information about users in the response. This parameter accepts a comma-separated list. Expand options include: *  `groups` includes all groups and nested groups to which the user belongs. *  `applicationRoles` includes details of all the applications to which the user has access.",
+                        "x-ms-summary": "Expand"
+                    }
+                ],
+                "security":
+                [
+                    {
+                        "Authorization": []
+                    }
+                ],
+                "responses":
+                {
+                    "200":
+                    {
+                        "description": "Returned if the request is successful.",
+                        "schema": {},
+                        "examples":
+                        {
+                            "application/json":
+                            {
+                                "self": "https://your-domain.atlassian.net/rest/api/3/user?accountId=5b10a2844c20165700ede21g",
+                                "key": "",
+                                "accountId": "5b10a2844c20165700ede21g",
+                                "name": "",
```

Comment:
@depfeife-msft Empty properties can be removed until unless they are impacting the functionality.